.github/release.yml

```diff
@@ -5,7 +5,6 @@ changelog:
       labels:
         - breaking-change
     - title: line-openapi updates
-      authors: renovate
```

Comment:
According to https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes, there is no config `changelog.categories[*].authors`